include/mapbox/vector_tile.hpp

```diff
@@ -72,8 +72,8 @@ class layer {
     std::string name;
     std::uint32_t version;
     std::uint32_t extent;
-    std::map<std::string, std::uint32_t> keysMap;
-    std::vector<std::reference_wrapper<const std::string>> keys;
+    std::multimap<std::string, std::uint32_t> keysMap;
```

Comment:
Should we handle duplicate values in a similar way? Spec has `SHOULD NOT` for the `values` list:

_The set of values SHOULD NOT contain two or more values of the same type which are byte-for-byte identical._

Reply:
The `keysMap` is introduced to do fast lookup, due to the nature of `map`, duplicate values will be excluded. However, the values are saved as `std::vector<protozero::data_view> values`, so duplicate values are actually allowed.